test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -330,9 +330,11 @@ static void test_out_in_common(odp_bool_t ah,
 			       odp_auth_alg_t auth,
 			       const odp_crypto_key_t *auth_key,
 			       const odp_crypto_key_t *cipher_key_extra,
-			       const odp_crypto_key_t *auth_key_extra)
+			       const odp_crypto_key_t *auth_key_extra,
+			       ipsec_test_flags *flags)
```

Comment:
If there is this new flags parameter, would it not make sense to include the ah versus esp flag in it too? And I am not sure if the flags parameter should be a struct pointer like that or if it would be simpler here to just have a bitmap of flags defined as ENUM.

Reply:
Yes. It makes sense to include AH vs ESP also. I'm also considering using an ENUM instead. I've added one more test case (stats of proto error). Please have a look at it when I push next version. In general, can I assume that you are okay with one framework doing all tests based on flags?

Reply:
> In general, can I assume that you are okay with one framework doing all tests based on flags?

Yes, as I suppose it avoids having to duplicate the same code multiple times. Just make sure that it is clear from test output what exactly failed in a test failure.